contrib/environment-to-ini/environment-to-ini.go

```diff
@@ -137,6 +149,21 @@ func runEnvironmentToIni(c *cli.Context) error {
 				continue
 			}
 		}
+		if isFileBased {
+			isFile, err := util.IsFile(value)
+			if err != nil {
+				log.Fatal("Unable to check if %s is a file. Error: %v", value, err)
+			}
+			if isFile {
+				if content, err := os.ReadFile(value); err == nil {
+					value = string(content)
+				} else {
+					log.Fatal("Failed to load value from file '%s': %v", value, err)
+				}
+			} else {
+				log.Fatal("File '%s' not found", value)
+			}
+		}
```

Comment:
```suggestion
		if isFileBased {
			if content, err := os.ReadFile(value); err == nil {
				value = string(content)
			} else {
				log.Fatal("Failed to load value from file %q: %v", value, err)
			}
		}
```

I guess this is enough?